ao-build-config.yml

```diff
@@ -0,0 +1,9 @@
+# ao-dev-cli options
+stack_size: 65536 # 64kb
+initial_memory: 1048576 # 1mb stack size is included in initial memory, along with heap
```

Comment:
Nits:
- use KiB, MiB units in comments.
- `1mb stack size is included in initial memory, along with heap` -> `1 MiB. Stack size utilizes a portion of this initial memory, and the rest is for the heap and statics`

Reply:
[091b4fc](https://github.com/ar-io/ar-io-ant-process/pull/54/commits/091b4fcb8de8f2f1a342d2578d225f6af707f2c6)